src/Fetch_Abort.res

```diff
@@ -1,10 +1,39 @@
+module AbortEvent = {
```

Comment:
This should just be named `Event` here, to be consistent with the other submodules, and aliased as `AbortEvent` in `Fetch`.